phpseclib/Net/SSH1.php

```diff
@@ -547,7 +547,7 @@ private function connect()
     {
         $this->fsock = @fsockopen($this->host, $this->port, $errno, $errstr, $this->connectionTimeout);
         if (!$this->fsock) {
-            throw new \RuntimeException(rtrim("Cannot connect to $host. Error $errno. $errstr"));
+            throw new \RuntimeException(rtrim("Cannot connect to $this->host. Error $errno. $errstr"));
```

Comment:
Interesting. It looks like this is only an issue in the master branch.

When I get around to revamping the API for SSH I will probably be deleting SSH1.php FWIW but, in the interim, I'm ok with having the issue fixed. Nice find!